CAP/gap/DerivedMethods.gi

```diff
@@ -677,6 +677,17 @@ AddDerivationToCAP( IsIsomorphism,
 end : CategoryFilter := IsAbelianCategory,
       Description := "IsIsomorphism by deciding if it is a mono and an epi" );
 
+##
+AddDerivationToCAP( IsIsomorphism,
+                    [ [ IsSplitMonomorphism, 1 ],
+                      [ IsSplitEpimorphism, 1 ] ],
+                 
+  function( morphism )
+    
+    return IsSplitMonomorphism( morphism ) and IsSplitEpimorphism( morphism );
+    
+end : Description := "IsIsomorphism by deciding if it is a split mono and a split epi" );
+
```

Comment:
We can merge this for now, but those derivations will soon be automatically generated from the predicates.